src/bootman/kernel.c

```diff
@@ -592,36 +592,26 @@ bool boot_manager_install_kernel_internal(const BootManager *manager, const Kern
         autofree(char) *base_path = NULL;
         autofree(char) *initrd_target = NULL;
         const char *initrd_source = NULL;
-        bool is_uefi = false;
-        autofree(char) *efi_boot_dir = NULL;
+        int is_uefi = (manager->bootloader->get_capabilities(manager) & BOOTLOADER_CAP_UEFI);
```

Comment:
Why int? Should be a `bool` to make it clear that we're set to true if the capabilites export `BOOTLOADER_CAP_UEFI`

Reply:
with `int` the expression is shorter (no need to compare with the value of the bit flag).

Reply:
@busykai I may be a little confused are you saying eventually it may be useful to keep the is_uefi around as a set of bit flags (hense int would make more sense)? Otherwise I agree with @ikeydoherty and think the intended meaning is clearer with bool (the variable name indicates true or false to me too).

Reply:
i'll just change this to `bool`.